app/frontend/src/cljs/teet/project/project_navigator_view.cljs

```diff
@@ -433,7 +433,9 @@
         (when (and
                 (common-controller/feature-enabled? :vektorio)
                 (some? (:vektorio/project-id project)))
-          [common/vektorio-link {:href (common-controller/query-url :vektorio/instant-login project)
+          [common/vektorio-link {:href (common-controller/query-url
+                                         :vektorio/instant-login {:db/id (:db/id project)
```

Comment:
could be done with (common-controller/query-url :vektorio/instant-login (select-keys project [:db/id :vektorio/project.id])) as well.